weave/trace_server/objects_query_builder.py

```diff
@@ -311,7 +324,7 @@ def make_objects_val_query_and_parameters(
     project_id: str, object_ids: list[str], digests: list[str]
 ) -> tuple[str, dict[str, Any]]:
     query = """
-        SELECT object_id, digest, any(val_dump)
```

Comment:
This was causing us to return null values for objects that were deleted and then recreated. Sometimes we selected the val_dump from the deletion event row.